test/Common/Microsoft.Test.OData.DependencyInjection/TestContainerBuilder.cs

```diff
@@ -5,48 +5,18 @@
 //---------------------------------------------------------------------
 
 using System;
-using System.Diagnostics;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.OData;
 using ServiceLifetime = Microsoft.Extensions.DependencyInjection.ServiceLifetime;
 
 namespace Microsoft.Test.OData.DependencyInjection
 {
-    public class TestContainerBuilder : IContainerBuilder
+    public class TestContainerBuilder 
     {

```

Comment:
I think we can get rid of his entire file altogether since we no longer use `IContainerBuilder`.